### ISSUE TITLE
Saving a file does not report failures

### ISSUE THREAD

**Issue by adrianheine** [deltachat/deltachat-desktop#2474]
- Operating System (Linux/Mac/Windows/iOS/Android): Debian GNU/Linux
- Delta Chat Version: 1.22.2
- Expected behavior: An error is reported
- Actual behavior: No feedback whatsoever is provided
- Steps to reproduce the problem: Try to save a file in a location that's not writable
- Screenshots:
- Logs:

```
Error occurred in handler for 'saveFile': [Error: EACCES: permission denied, copyfile '/home/XXX/.config/DeltaChat/accounts/XXX/dc.db-blobs/XXX.jpg' -> '/x'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'copyfile',
  path: '/home/XXX/.config/DeltaChat/accounts/XXX/dc.db-blobs/XXX.jpg',
  dest: '/x'
}
```

**Comment by IrvinLara9** [comment]
I would like to work on this issue!

**Comment by Simon-Laux** [comment]
was solved by #2479

useful hint: you can write `closes #issue-number` in your pr description so the issue gets automatically closed once the pr is merged.
